delatore/json2mdwn.py

```diff
@@ -2,6 +2,12 @@
 import re
 
 
+def _normalize_key(src: str):
+    if src.isupper():
```

Comment:
Maybe skip only `.title()` for upper case keys? Still replacing `_` with ` ` for them

Reply:
No